deps/minizip/zip.c

```diff
@@ -1862,7 +1851,7 @@ int Write_EndOfCentralDirectoryRecord(zip64_internal* zi, uLong size_centraldir,
    return err;
 }
 
-int Write_GlobalComment(zip64_internal* zi, const char* global_comment)
+static int Write_GlobalComment(zip64_internal* zi, const char* global_comment)
 {
   int err = ZIP_OK;
   uInt size_global_comment = 0;
```

Comment:
Yea, I've actually been meaning to take 10 minutes and move it into `src/minizip.c`, and build `src/libminizip.la`. The `deps` directory made some sense when we also built AntTweakBar, but with just minizip in there, it's kinda silly. So I'm in favor of moving it into `src` and treating it just like, say, `libmatrix.la` -- with astyle and other cleanup.